lib/config/config.js

```diff
@@ -200,11 +200,17 @@ module.exports = function (fs, path, url, convict) {
         env: 'IP_BLOCKLIST_LOGONLY'
       },
       lists: {
-        doc: 'A array of ip blocklist file paths',
+        doc: 'An array of ip blocklist file paths',
         format: Array,
         default: [],
         env: 'IP_BLOCKLIST_FILES'
       },
+      listsOverrideRules: {
+        doc: 'An array of true/false values that map directly the the lists property and determines if that specific list should be blocked',
+        format: Array,
+        default: [],
+        env: 'IP_BLOCKLIST_FILES_BLOCKING'
+      },
```

Comment:
This approach will work, but it seems to make the code hard to follow, and it's hard to think of a good name for the config option that explains how to use it.  I wonder if it would be simpler to just have to separate lists of files, one that is log-only and one that actually blocks, something like:

```
ipBlocklist: {
  enable: true,
  lists: ["/this/one/blocks.netset"],
  logOnlyLists: ["/this/one/logs.netset"]
}
```